src/js/components/landing/recentActivity/SubmissionLink.jsx

```diff
@@ -18,10 +19,25 @@ const defaultProps = {
 }
 
 export default class SubmissionLink extends React.Component {
+	constructor(props) {
+		super(props);
+		this.state = {
+			page: '#/'
```

Comment:
Can the page ever be redirected to the home page? If not, we can just have this preset to an empty string instead.